### PR TITLE
Feature/fdom/facebook login

### PR DESCRIPTION
**Pull request by Fdom92**
**Short description of this feature:**

Adds a facebook login button on the page with this funtionality

- on success show up an alert with the user name
- on fail show up an error message
- on cancel show up an alert with login canceled
- on logout show up an alert to notify logout

### CODE REVIEW

File: src/pages/login.js [hunk 1]
```diff
@@ -1,16 +1,26 @@
-import {View, StyleSheet} from 'react-native';
-import React, {Component, PropTypes} from 'react';
-import {routes} from '../routes';
-import core_styles from '../styles/core-styles';
-import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome';
-import {Sae} from 'react-native-textinput-effects';
-import {Button} from 'react-native-material-ui';
-import theme from '../styles/ui-theme';
+import React, { Component, PropTypes } from 'react';
```
Comment (JorgeGarciaIrazabal) [inline src/pages/login.js:1]: Why this blank lines in the imports?

Is the logic:
- react dependencies:
- 3rd party libraries 
- our dependencies
??

File: src/pages/login.js [hunk 2]
```diff
@@ -35,7 +45,27 @@ class Login extends Component {
           autoCapitalize={'none'}
           secureTextEntry={true}
         />
-        <Button primary text="Login" />
+        <View style={core_styles.facebookBtn}>
```
Comment (JorgeGarciaIrazabal) [inline src/pages/login.js:48]: maybe `facebookBtn` is too specific to be in core_styles, We could have it in this same file or in another file for the login page

**Review by JorgeGarciaIrazabal — APPROVED** [review]
Works perfectly in my computer!!

File: src/pages/login.js (hunk 1 above)
Comment (Fdom92) [reply src/pages/login.js:1]: I usually organize it like that, react dependencies and then white space, then 3rd libraries and white space and finally our dependencies.

File: src/pages/login.js (hunk 2 above)
Comment (Fdom92) [reply src/pages/login.js:48]: Yes,  totally agree! Just for try this but forget move it to another file

**Pull request merged**